pynicotine/userbrowse.py

```diff
@@ -93,7 +92,7 @@ def browse_local_shares(self, path=None, share_type="buddy", new_request=False):
         username = config.sections["server"]["login"] or "Default"
 
         if username not in self.user_shares or new_request:
-            msg = core.shares.get_compressed_shares_message(share_type)
+            msg = core.shares.compressed_shares.get(share_type)
```

Comment:
Set default `share_type="trusted"` instead of `"buddy"`

Alternatively, get our own `is_buddy` and `is_trusted` state to determine which shares list to get in the case of manually entered self browse shares request.

Reply:
Local shares will now update based on our own permission level, unless we've explicitly browsed a specific share type via the menu.